curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b61584def8fa2ebcc259e0.md

```diff
@@ -7,7 +7,17 @@ dashedName: step-19
 
 # --description--
 
-You need to split your `str` into individual characters. Remember that strings have a `.split()` method, and if you pass an empty string it will split on every character.
+You need to split your `str` into individual characters. You can use the <dfn>split()</dfn> method to do this.
+
+The `split()` method splits a string into an array of substrings, and returns the new array. You can pass in an optional separator which tells the computer where each split should happen.
+
+Here is an example of how to use it:
```

Comment:
```suggestion
For example, passing an empty string into the `split` method will split the string into an array of individual characters.
```

I think it is helpful to state in words what the example is demonstrating, at least to the extent possible.